source/manual/readmes.html.md

```diff
@@ -74,10 +74,12 @@ A good example:
 
 > GOV.UK filtered search of public content
 
-## Things we shouldn't put in the README
+## CONTRIBUTING.md
 
-- Contributors - we can use GitHub's native graphing tools for this
-- Contribution guidelines - use GitHub's [CONTRIBUTING.md](https://help.github.com/articles/setting-guidelines-for-repository-contributors/) guidelines for this
-  - When writing this document, always link to the [GOV.UK pull request style guide](https://github.com/alphagov/styleguides/blob/master/pull-requests.md)
-  - Document any weirdness here (eg when to use Cucumber over something else)
-- Full licence - again, follow [GitHub convention](https://help.github.com/articles/open-source-licensing/#where-does-the-license-live-on-my-repository)
+**Don't add this unless the repository gets a lot of external contributions.**
+
+Use GitHub's [CONTRIBUTING.md](https://help.github.com/articles/setting-guidelines-for-repository-contributors/) guidelines for this. Always link to the [GOV.UK pull request style guide](https://github.com/alphagov/styleguides/blob/master/pull-requests.md).
+
+## LICENCE.md
```

Comment:
Is it `LICENCE.md` or simply `LICENCE`? E.g. Content Publisher has no file extension: https://github.com/alphagov/content-publisher/blob/main/LICENCE

Reply:
Oh dear, another can of worms. Analysis of the projects I have locally:

- 5 have `LICENSE.md`
- 4 have `LICENCE.md`
- 42 have `LICENCE`
- 24 have `LICENSE`

🙄. I've amended to `LICENCE` because it reflects the majority and the plaintext content of the file. I've added a task on [the follow-up card](https://trello.com/c/VU3R1B8X/240-make-govuk-readmes-consistent-with-the-gds-way) to decide if it's worth renaming the others.

As a side-note, quite a few of our repos also have a `CONTRIBUTING.md` (happily there are no other variants!). I've added another follow-up task to decide if we want to delete them (all).